src/common/components/repositories-list/index.js

```diff
@@ -23,9 +23,13 @@ const SNAP_NAME_NOT_REGISTERED_ERROR_CODE = 'snap-name-not-registered';
 export class RepositoriesListView extends Component {
 
   fetchAuthData(authStore) {
-    if (authStore.authenticated === null) {
+    // check if authenticated and hasShortNamespace haven't been set yet,
+    // so are null or undefined
+    if (authStore.authenticated === null ||
+        typeof authStore.authenticated === 'undefined') {
```

Comment:
This seems like an oddly verbose way to write `authStore.authenticated === undefined` (which as far as I can tell works fine).